x/spec/types/types.go

```diff
@@ -35,10 +35,11 @@ const (
 const (
 	GET_BLOCKNUM                = "getBlockNumber"
 	GET_BLOCK_BY_NUM            = "getBlockByNumber"
+	GET_CHAIN_ID                = "getChainId"
```

Comment:
this changed to an enum in the latest version make sure to make adjustments with the 701 PR